python-frontend/typeshed_serializer/resources/custom/urllib2/__init__.pyi

```diff
@@ -0,0 +1,67 @@
+import ssl
+from typing import Any, Dict, List, Optional, Tuple, Type, Union
+
+class Request(object):
+    host: str
+    port: str
+    data: str
+    headers: Dict[str, str]
+    unverifiable: bool
+    type: Optional[str]
+    origin_req_host = ...
+    unredirected_hdrs: Dict[str, str]
+    timeout: Optional[float]  # Undocumented, only set after __init__() by OpenerDirector.open()
+    def __init__(
+        self,
+        url: str,
+        data: Optional[str] = ...,
+        headers: Dict[str, str] = ...,
+        origin_req_host: Optional[str] = ...,
+        unverifiable: bool = ...,
+    ) -> None: ...
+    def __getattr__(self, attr): ...
+    def get_method(self) -> str: ...
+    def add_data(self, data) -> None: ...
+    def has_data(self) -> bool: ...
+    def get_data(self) -> str: ...
+    def get_full_url(self) -> str: ...
+    def get_type(self): ...
+    def get_host(self) -> str: ...
+    def get_selector(self): ...
+    def set_proxy(self, host, type) -> None: ...
+    def has_proxy(self) -> bool: ...
+    def get_origin_req_host(self) -> str: ...
+    def is_unverifiable(self) -> bool: ...
+    def add_header(self, key: str, val: str) -> None: ...
+    def add_unredirected_header(self, key: str, val: str) -> None: ...
+    def has_header(self, header_name: str) -> bool: ...
+    def get_header(self, header_name: str, default: Optional[str] = ...) -> str: ...
+    def header_items(self): ...
```

Comment:
I have kept the Request and urlopen stubs even if they are not directly linked to the issue but I think it would be nice to not raise the TPs on the Security side because of a lucky match of the FQNs.